map_model/src/objects/road.rs

```diff
@@ -434,17 +436,6 @@ impl Road {
         result
     }
 
-    /// lane must belong to this road. Offset 0 is the centermost lane on each side of a road, then
```

Comment:
I just moved this up to be next to `offset`. I thought I could consolidate, but they calculate different things.